internal/cloudapi/v2/openapi.v2.yml

```diff
@@ -548,6 +569,21 @@ components:
             items:
               $ref: '#/components/schemas/Error'
 
+    DistributionList:
+      properties:
+        map:
+          type: object
+          additionalProperties:
+            map:
+              type: object
+              additionalProperties:
+                map:
+                  type: object
+                  additionalProperties:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/BlueprintRepository'
```

Comment:
This is IMHO impossible to read and determine the meaning of returned values. I'd suggest at least adding a description to each property.